src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Row,
+  Col,
+  Space,
+  Collapse,
+  Skeleton,
+  Spin,
+  Radio,
+} from 'antd';
+import { useRouter } from 'next/router';
+import { useSelector, useDispatch } from 'react-redux';
+import { Vega } from 'react-vega';
+import DimensionsRangeEditor from '../components/DimensionsRangeEditor';
+import TitleDesign from '../components/TitleDesign';
+import AxesDesign from '../components/AxesDesign';
+import FontDesign from '../components/FontDesign';
+import LegendEditor from '../components/LegendEditor';
+import generateSpec from '../../../../../utils/plotSpecs/generateFrequencySpec';
+import Header from '../components/Header';
+import isBrowser from '../../../../../utils/environment';
+import {
+  updatePlotConfig,
+  loadPlotConfig,
+} from '../../../../../redux/actions/plots/index';
+import PlatformError from '../../../../../components/PlatformError';
+import loadCellSets from '../../../../../redux/actions/cellSets/loadCellSets';
+
+const { Panel } = Collapse;
+const plotUuid = 'frequencyPlotMain';
+const plotType = 'frequency';
+const route = {
+  path: 'frequency',
+  breadcrumbName: 'Frequency plot',
+};
+
+const frequencyPlot = () => {
+  const dispatch = useDispatch();
+  const config = useSelector((state) => state.plots[plotUuid]?.config);
+  const { loading, error } = useSelector((state) => state.cellSets);
+  const properties = useSelector((state) => state.cellSets.properties);
+  const router = useRouter();
+  const { experimentId } = router.query;
+  const [plotSpec, setPlotSpec] = useState({});
+  const [plotData, setPlotData] = useState();
+
+  useEffect(() => {
+    if (!experimentId || !isBrowser) {
+      return;
+    }
+    dispatch(loadCellSets(experimentId));
+
+    dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
+  }, [experimentId]);
+
+  useEffect(() => {
+    if (!config || Object.keys(properties).size === 0) {
+      return;
+    }
+    setPlotSpec(generateSpec(config));
+    setPlotData(generateData());
+  }, [config, properties]);
+
+  const generateData = () => {
+    const data = [];
+    if (!loading) {
+      let i = 0;
+      const sum = properties['louvain-0'].cellIds.size
+        + properties['louvain-1'].cellIds.size
+        + properties['louvain-2'].cellIds.size;
+      let value;
+      for (i = 0; i < 3; i += 1) {
+        if (config.plotType === 'count') {
+          value = properties[`louvain-${i}`].cellIds.size;
+        } else {
+          value = (properties[`louvain-${i}`].cellIds.size / sum) * 100;
+        }
+        data.push({
+          x: 1,
+          y: value,
+          c: `Cluster ${i}`,
+        });
+      }
```

Comment:
remove the hardcoding of `louvain` in here and make it work for all clusters, not only for the first 3